topics/fair/tutorials/fair-ego/tutorial.md

```diff
@@ -0,0 +1,217 @@
+---
+layout: tutorial_hands_on
+title: FAIR and RDM from a selfish perspective
+abbreviations:
+  FAIR: Findable, Accessible, Interoperable, Reusable
+  GTN: Galaxy Training Network
+  RDM: Research Data Management
+  DPM: Data Management Plan
+level: Introductory
+zenodo_link: ''
+questions:
+- Why should other researchers adhere to FAIR principles?
+- How can I profit from good RDM?
+objectives:
+- Learn about the benefits of good data management
+time_estimation: "30M"
+key_points:
+- RDM can lead save time and money.
+tags:
+- fair
+- gtn
+- rdm
+contributions:
+  authorship:
+    - korneelhens
+    - nomadscientist
+  funding:
+    - elixir-fair-data
+subtopic: fair-data
+
+requirements:
+  - type: "internal"
+    topic_name: fair
+    tutorials:
+      - fair-intro
+   
+draft: true
+
+---
+
+
+# Introduction
+
+**You** are obviously too important to bother with this research data management ({RDM}) nonsense. Just copy the data management plan ({DPM})from a previous grant application and stop wasting time.
```

Comment:
```suggestion
**You** are obviously too important to bother with this {RDM} nonsense. Just copy the {DMP} from a previous grant application and stop wasting time.
```

you can just use `{..}` and it'll properly expand it also the first time :) (also this is a funny introduction, it's nice! :D)